question 5 is out of scope for now.

sujeet472 opened the pull request:
All questions are added and created a PR. Review the questions so that I can add it to main.

sujeet472 committed:
question 5 is same to others

samu80 commented on sujeet/question2.txt:
```diff
@@ -0,0 +1,6 @@
+Q2. How to set multiple remote repositories for the same project? Explain the use case. you should have a working demo of it.
+
+Ans: git remote add origin (url);
+     git add upstream (url)
```
Please add stepwise commands for how to create multiple repos on different platforms

samu80 commented on sujeet/question3.txt:
```diff
@@ -0,0 +1,5 @@
+
+Ans: git rebase -i HEAD~4
+
+or git rebase head~4 (int this option you will not get the option to chose the comnit);
```
Add interactive terminal commands also

sujeet472 committed:
Update question2.txt

Command added for github and gitlabs for pushing a change of a file

sujeet472 committed:
Update question3.txt

This process is called squashing and have added the step wise commands for the same
